packages/generators/generators/package.json

```diff
@@ -30,13 +30,18 @@
   "main": "lib/index.js",
   "dependencies": {
     "@sindresorhus/slugify": "1.1.0",
+    "@strapi/icons": "1.6.3",
     "@strapi/typescript-utils": "4.6.1",
     "@strapi/utils": "4.6.1",
+    "axios": "1.2.2",
     "chalk": "4.1.2",
     "fs-extra": "10.0.0",
     "node-plop": "0.26.3",
     "plop": "2.7.6",
-    "pluralize": "8.0.0"
+    "pluralize": "8.0.0",
+    "prop-types": "^15.7.2",
+    "react": "^17.0.2",
+    "react-router-dom": "5.3.4"
```

Comment:
These are an interesting one too. The `files` are just copied over when someone makes a new plugin, how come you think their deps need to be declared? 😄

Reply:
Done.